.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Bug report'
+labels: bug
+---
+
+## Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
+
+## Current Behavior
+
+What is the current behavior?
+
+### Steps to Reproduce
+
+Please provide detailed steps for reproducing the issue.
+
+1. step 1
+2. step 2
+3. ...
+
+### Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
```

Comment:
I would add a request to post contents of `package.json` if possible (no senstive info, for example). This way we can have full information about the context.

Reply:
Done.